install/updateVersion.sh

```diff
@@ -103,24 +109,35 @@ function check_git_status() {
 # Generated merge yaml files for easy installation
 function merge_files() {
   SRC=$TEMP_DIR/templates
+  DEST=$ROOT/install/kubernetes
   AUTH_SRC=$SRC/istio-auth
-  ISTIO=$ROOT/install/kubernetes/istio.yaml
-  ISTIO_AUTH=$ROOT/install/kubernetes/istio-auth.yaml
-  ISTIO_AUTH_WITH_CLUSTER_CA=$ROOT/install/kubernetes/istio-auth-with-cluster-ca.yaml
+  ISTIO=$DEST/istio.yaml
+  ISTIO_CLUSTER_WIDE=$DEST/istio-cluster-wide.yaml
+  ISTIO_AUTH=$DEST/istio-auth.yaml
 
   echo "# GENERATED FILE. Use with Kubernetes 1.7+" > $ISTIO
```

Comment:
make sure the header is applied to all generated files; we have people editing the generated files
(in theory we shouldn't even have the generated files in source control but ... different battle)

Reply:
?.. istio.yaml is the only file that we ship.